test/jdk/javax/swing/JMenuItem/bug4966168.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2004, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*  @test
+    @bug 4966168
+    @summary JInternalFrame not serializable in Motif & GTK L&F
```

Comment:
I dont see the test being tested for Motif & GTK which are not default L&F...Probably we should iterate for all L&F just to be sure..
Also, all `throw new Error` should ideally be changed to `throw new RuntimeException` for all tests in this PR..

Reply:
After modifying the test to run in all L&F, CI failure observed for linux and linux-aarch machine with the exception 
`javax.swing.UnsupportedLookAndFeelException: [GTK look and feel - com.sun.java.swing.plaf.gtk.GTKLookAndFeel] not supported on this platform`.

On my local machine (Oracle Linux) test passed for all LAFs.

Reply:
It probably needs headful tag for that.
Also, you should `continue` for UnsupportedLookAndFeelException...and not fail, just print a message and continue to next L&F

Reply:
Updated for all LAFs.